File: node-express/src/services/userService.js
const User = require('../models/userModel');

const users = [
  new User(1, 'Murilo Henrique', 'murilo@email.com'),
  new User(2, 'Ana Silva', 'ana@email.com')
];

function listUsers() {
  return users;
}

function getExampleUser() {
  return users[0];
}

module.exports = {
  listUsers,
  getExampleUser
};

File: java-springboot/src/main/java/com/example/demo/services/UserService.java
package com.example.demo.services;

import com.example.demo.models.UserModel;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
public class UserService {
    public List<UserModel> listUsers() {
        return List.of(
                new UserModel(1L, "Murilo Henrique", "murilo@email.com"),
                new UserModel(2L, "Ana Silva", "ana@email.com")
        );
    }
}
